Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+            sh:path scenario:coordinateSystem ],
+        [ skos:example "EPSG xyz" ;
+            sh:datatype xsd:string ;
+            sh:description "Ellipsodial height or orthometric height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of heightSystem failed!"@en ;
+            sh:name "heightSystem"@en ;
+            sh:order 1 ;
+            sh:path scenario:heightSystem ] ;
+    sh:targetClass scenario:GeodeticReferenceSystem .
+
+scenario:BoundingBoxShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of yMin failed!"@en ;
+            sh:name "yMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path scenario:yMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of yMax failed!"@en ;
+            sh:name "yMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 3 ;
+            sh:path scenario:yMax ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMin failed!"@en ;
+            sh:name "xMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path scenario:xMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMax failed!"@en ;
+            sh:name "xMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path scenario:xMax ] ;
+    sh:targetClass scenario:BoundingBox .
+
+scenario:Coordinate2DShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of x failed!"@en ;
+            sh:name "x"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path scenario:x ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of y failed!"@en ;
+            sh:name "y"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path scenario:y ] ;
+    sh:targetClass scenario:Coordinate2D .
+    
+scenario:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "State of project area"@en ;
+            sh:message "Validation of state failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "state"@en ;
+            sh:order 1 ;
+            sh:path scenario:state ],
+        [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "nan"@en ;
+            sh:message "Validation of city failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "city"@en ;
+            sh:order 3 ;
+            sh:path scenario:city ],
+        [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "Region of project area"@en ;
+            sh:message "Validation of region failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "region"@en ;
+            sh:order 2 ;
+            sh:path scenario:region ],
+        [ skos:example "Germany, GER" ;
```

Comment:
cant you use the address shape which is already in the gaia-x vocabulary?

<http://www.w3.org/2006/vcard/ns#Address>

if yes you can add it as a shape with following approach:
```
[ sh:minCount 1 ;
            sh:node <http://www.w3.org/2006/vcard/ns#Address> ;
            sh:path environmentmodel:address;
            sh:order 5 ] ;
```
Or you could use PostalAddressShape:
```
[ sh:minCount 1 ;
            sh:node gx:PostalAddressShape ;
            sh:path environmentmodel:address;
            sh:order 5 ] ;
```
But this has only one property for a  ISO 3166-2 based string.

Reply:
Would prefere a reference to ISO 3166-2 (e.g., for Germany ISO 3166-2:DE -> DE-BY). Therefore, we can also refer to gx-shape, right?

Reply:
Should be solved in the georef-shape. @3Dbastian, please take a look at it.